docs/sources/enterprise/saml.md

```diff
@@ -46,6 +46,7 @@ The table below describes all SAML configuration options. Continue reading below
 | ---------------------------------------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
 | `enabled`                                                  | No       | Whether SAML authentication is allowed                                                                                                                                                              | `false`       |
 | `single_logout`                                            | No       | Whether SAML Single Logout enabled                                                                                                                                                                  | `false`       |
+| `allow_sign_up`                                            | No       | Whether to allow new Grafana user creation through SAML login. If set to `false`, then only existing Grafana users can log in with SAML.                                                            | `true`        |
```

Comment:
```suggestion
| `allow_sign_up`                                            | No       | Whether to allow new Grafana user creation through SAML login. If set to `false`, only existing Grafana users can log in with SAML                                                           | `true`        |
```

Reply:
I have had many discussions about the If - then. Like you, and as I showed Google style guide examples, we do not need to be rigid. The consensus was otherwise (this was at least a year ago). I do not know if opinions have changed.